mopidy_spotify/library.py

```diff
@@ -63,20 +63,13 @@ def _lookup_album(self, sp_link):
     def _lookup_artist(self, sp_link):
         sp_artist = sp_link.as_artist()
         sp_artist_browser = sp_artist.browse(
-            type=spotify.ArtistBrowserType.NO_TRACKS)
+            type=spotify.ArtistBrowserType.FULL)
```

Comment:
Maybe they mean to say there's no single magic sp_artist function to just get albums (rather than all the other artist stuff too)? Or the note is just outdated.